charts/platform-system/values.yaml

```diff
@@ -145,6 +145,13 @@ components:
           source: {}
         # values override HelmRelease values
         values: {}
+        components:
+          externalAuth:
+            # Enable only after `platform-auth` installed and its traefik proxy is running
+            enabled: false
+            name: "platform-auth-proxy"
+            service: "platform-auth-proxy.platform-auth-proxy.svc.cluster.local"
```

Comment:
Is this from the config chart? Do we want to template this out? No opinion here just asking out loud

Reply:
This is sort of an output from `platform-auth`. I don't like this, but it's kinda the only way to get it in there. If most naming defaults are left untouched the new values should be accurate:
`auth-proxy.traefik-proxy.svc.cluster.local`